src/com/palmergames/bukkit/towny/object/economy/BankAccount.java

```diff
@@ -111,6 +111,14 @@ protected boolean addMoney(double amount) {
 		return TownyEconomyHandler.add(getName(), amount, world);
 	}
 
+	@Override
+	public boolean canPayFromHoldings(double amount) {
+		if (isBankrupt())
+			return addDebt(amount);
```

Comment:
This should use the getDebtCap() instead of directly adding debt

Reply:
Good catch.